teeio-validator/library/cxl_ide_test_lib/test_case/ide_km_common.c

```diff
@@ -106,6 +115,7 @@ bool cxl_setup_ide_stream(void *doe_context, void *spdm_context,
     TEEIO_DEBUG((TEEIO_DEBUG_ERROR, "libspdm_get_random_number for rx_key_buffer failed.\n"));
     return false;
   }
+  rx_key_buffer.iv[1] = 1;
```

Comment:
`iv[1] = 1` ?

Reply:
It is not needed because default IV is used.